src/js/window_pen.js

```
@@ -529,11 +532,27 @@ export class PenSystem extends ToolWindow {
                     e.clientY
                 )
                 if (distance < stabilizer_value) {
-                    //console.log('補正', distance);
+                    // 確定点としては採用しないが、整数xyが変わったタイミングで暫定描画して
+                    // ペン直下に「暫定的なベジェ終端」を表示する（ラグの体感を軽減）
+                    if (this.axpObj.isDrawing && !this.axpObj.isDrawCancel) {
+                        const intDist = Math.floor(distance);
```

ESLint

[error] 'intDist' is assigned a value but never used.

(no-unused-vars)